community/rfcs/24-08-02-OPEA-AIAvatarChatbot.md

```diff
@@ -0,0 +1,224 @@
+    classDef orchid fill:#C26DBC,stroke:#ADD8E6,stroke-width:2px,fill-opacity:0.5
+    classDef invisible fill:transparent,stroke:transparent;
+    style AvatarChatbot-Megaservice stroke:#000000
+
+    %% Subgraphs %%    
+    subgraph AvatarChatbot-Megaservice["AvatarChatbot Megaservice"]
+        direction LR
+        ASR([ASR<br>3001]):::blue
+        LLM([LLM 'text-generation'<br>3007]):::blue
+        TTS([TTS<br>3002]):::blue
+        animation([Animation<br>3008]):::blue
+    end
+    subgraph UserInterface["User Interface"]
+        direction LR
+        invis1[ ]:::invisible
+        USER1([User Audio Query]):::orchid
+        USER2([User Image/Video Query]):::orchid
+        UI([UI server<br>]):::orchid
+    end
+    subgraph ChatQnA GateWay
+        direction LR
+        invis2[ ]:::invisible
+        GW([AvatarChatbot GateWay<br>]):::orange
+    end
+    subgraph  
+        direction LR
+        X([OPEA Microservice]):::blue
+        Y{{Open Source Service}}:::thistle
+        Z([OPEA Gateway]):::orange
+        Z1([UI]):::orchid
+    end
+
+    %% Services %%    
+    WHISPER{{Whisper service<br>7066}}:::thistle
+    TGI{{LLM service<br>3006}}:::thistle
+    T5{{Speecht5 service<br>7055}}:::thistle
+    WAV2LIP{{Wav2Lip service<br>3003}}:::thistle
+
+    %% Connections %%
+    direction LR
+    USER1 -->|1| UI
+    UI -->|2| GW
+    GW <==>|3| AvatarChatbot-Megaservice
+    ASR ==>|4| LLM ==>|5| TTS ==>|6| animation
+
+    direction TB
+    ASR <-.->|3'| WHISPER
+    LLM <-.->|4'| TGI
+    TTS <-.->|5'| T5
+    animation <-.->|6'| WAV2LIP
+
+    USER2 -->|1| UI
+    UI <-.->|6'| WAV2LIP
+```
+
+
+#### Frontend UI
+The frontend UI is Gradio. User is prompted to upload either an image or a video as the avatar source. The user also asks his question verbally through the microphone by clicking on the "record" button. The AvatarChatbot backend processes the audio input and generates the response in the form of an animated avatar answering in its unique voice. The response is displayed on Gradio UI. User will be able to see the animated avatar speaking the response in real-time, and can interact with the avatar by asking more questions.
+<div style="display: flex; justify-content: space-between;">
+  <img src="assets/ui_latest_1.png" alt="alt text" style="width: 33%;"/>
+  <img src="assets/ui_latest_2.png" alt="alt text" style="width: 33%;"/>
+  <img src="assets/ui_latest_3.png" alt="alt text" style="width: 33%;"/>
+</div>
+
+### Real-time demo
+AI Avatar Chatbot Demo on Intel® Gaudi® 2, image input (top) and video input (down)
+<!-- [AI Avatar Chatbot Demo on Intel® Gaudi® 2, image input](https://intel.sharepoint.com/:v:/s/mlconsultingandsupport/EZa7vjON10ZCpMvE7U-SPMwBRXbVHqe1Ybsa-fmnXayNUA?e=f6FPsl)   
+[AI Avatar Chatbot Demo on Intel® Gaudi® 2, video input](https://intel.sharepoint.com/:v:/s/mlconsultingandsupport/ESMIcBseFTdIuqkoB7TZy6ABfwR9CkfV49TvTa1X_Jihkg?e=zMH7O7) -->
```

Comment:
@ctao456  We shouldn't have such an URL in our documentation (it's not accessible to the community). If that video is important, it should be put in a public location.